components/UploadedItems.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from "react";
+
+var items = 50;
+var finalURL = `https://archive.org/advancedsearch.php?q=bub.wikimedia+&fl%5B%5D=identifier&fl%5B%5D=publicdate&fl%5B%5D=title&sort%5B%5D=publicdate+desc&sort%5B%5D=&sort%5B%5D=&rows=${items}&page=1&output=json`;
+
+class UploadedItems extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      result: 0
+    }
+    this.fetchData = this.fetchData.bind(this);
+  }
+
+  fetchData() {
+    fetch(finalURL)
+      .then((response) => response.json())
+      .then((responseJson) => {
+        const resultyt = responseJson.response.numFound;
+        this.setState({ result: (resultyt-1) });
```

Comment:
Just curious, why are you decrementing the value?

Reply:
I think, in the uploads, it is also showing the account as 1 uploaded item. so, I need to decrement 1 from the total uploads

Reply:
No. I uploaded a file with that name when I was testing the API. Please remove the decrement.

Reply:
Also, use the `<Link>` component to add a link. Don't use the "a" tag. I have used `<Link>` in the code. You can see it from there. :)